### PR TITLE
allow symfony 4.0

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
This should allow Symfony 4.0
#SymfonyConHackday2017

### CODE REVIEW

**Comment by ElectricMaxxx** [comment]
@Soullivaneuh anything i can do?

File: .travis.yml [hunk 1]
```diff
@@ -37,6 +37,10 @@ matrix:
       env: SYMFONY_VERSION=3.1.*
     - php: 5.6
       env: SYMFONY_VERSION=3.2.*@dev
+    - php: 7.1
+      env: SYMFONY_VERSION=^3.4@dev
```
Comment (Soullivaneuh) [inline .travis.yml]: Please use `3.4.*`

File: .travis.yml [hunk 2]
```diff
@@ -37,6 +37,10 @@ matrix:
...
+    - php: 7.1
+      env: SYMFONY_VERSION=^4.0@dev
```
Comment (Soullivaneuh) [inline .travis.yml]: 4.0.*

File: .travis.yml [hunk 3]
```diff
@@ -37,6 +37,10 @@ matrix:
       env: SYMFONY_VERSION=3.1.*
     - php: 5.6
       env: SYMFONY_VERSION=3.2.*@dev
+    - php: 7.1
+      env: SYMFONY_VERSION=3.4.*
```
Comment (Soullivaneuh) [inline .travis.yml]: Well, you still need `@dev`. ^^

**Comment by ElectricMaxxx** [comment]
@Soullivaneuh i did the changes you whanted me to do.

**Comment by Soullivaneuh** [comment]
Travis CI seems to not like 4.0: https://travis-ci.org/Soullivaneuh/php-cs-fixer-styleci-bridge/jobs/306149952

**Comment by ElectricMaxxx** [comment]
it also depends on: https://github.com/matthiasnoback/SymfonyDependencyInjectionTest/pull/86

**Comment by ElectricMaxxx** [comment]
@Soullivaneuh can you restart builds, please.

File: .travis.yml [hunk 4]
```diff
@@ -36,7 +36,13 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=3.1.*
     - php: 5.6
-      env: SYMFONY_VERSION=3.2.*@dev
+      env: SYMFONY_VERSION=3.2.*
+    - php: 7.1
+      env: SYMFONY_VERSION=3.3.*
+    - php: 7.1
+      env: SYMFONY_VERSION=3.4.*@dev
```
Comment (Soullivaneuh) [inline .travis.yml]: You can remove `@dev` now. :+1:

File: .travis.yml [hunk 5]
```diff
@@ -36,7 +36,13 @@ matrix:
...
+    - php: 7.1
+      env: SYMFONY_VERSION=4.0.*@dev
```
Comment (Soullivaneuh) [inline .travis.yml]: Remove `@dev`

File: .travis.yml [hunk 6]
```diff
@@ -62,7 +68,7 @@ before_script:
 
 script:
   - composer validate
-  - phpunit -c phpunit.xml.dist --coverage-clover build/logs/clover.xml
+  - vendor/bin/simple-phpunit -c phpunit.xml.dist --coverage-clover build/logs/clover.xml
```
Comment (Soullivaneuh) [inline .travis.yml]: Didn't see that but why using simple-phpunit ?

And in this case, phpunit should be remove form the global require: https://github.com/Soullivaneuh/php-cs-fixer-styleci-bridge/pull/81/files#diff-354f30a63fb0907d4ad57269548329e3R61

**Pull request closed without merging**